content/en/tracing/guide/force_tracing.md

```diff
@@ -0,0 +1,178 @@
+---
+
+
+APM enables priority sampling by default to allow traces between two Datadog endpoints to be sampled together. This prevents trace sampling from removing segments of a distributed trace (i.e. ensures completeness) and helps removing unimportant ones. You can override this functionality to force keep a trace (critical transaction, debug mode, etc.) or force drop a trace (health checks, static assets, etc) by the agent and the server using forced tracing.
+
+Forced tracing should be done only before any context propagation. If this happens after the propagation of a context, the system can’t ensure that the entire trace is sampled properly.
+
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+Manually keep or drop a trace:
+
+```java
+import datadog.trace.api.DDTags;
+import datadog.trace.api.interceptor.MutableSpan;
+import datadog.trace.api.Trace;
+import io.opentracing.util.GlobalTracer;
+
+public class MyClass {
+    @Trace
+    public static void myMethod() {
+        // grab the active span out of the traced method
+        MutableSpan ddspan = (MutableSpan) GlobalTracer.get().activeSpan();
+
+        // ask the sampler to keep the current trace
+        ddspan.setTag(DDTags.MANUAL_KEEP, true);
+
+        // ask the sampler to drop the current trace
+        ddspan.setTag(DDTags.MANUAL_DROP, true);
+
+        // method impl follows
+    }
+}
+```
+
+[1]: /tracing/languages/java/#configuration
+{{% /tab %}}
+{{% tab "Python" %}}
+
+Manually keep or drop a trace:
+
+```python
+span = tracer.current_span()
+
+# Always keep this trace
+span.set_tag('manual.keep')
+
+# Always drop this trace
+span.set_tag('manual.drop')
+```
+
+{{% /tab %}}
+{{% tab "Ruby" %}}
+
+Manually keep or drop a trace:
+
+```ruby
+# Always keep this trace
+span.set_tag('manual.keep')
+
+# Always drop this trace
+span.set_tag('manual.drop')
+```
+{{% /tab %}}
+{{% tab "Go" %}}
+
+Manually keep or drop a trace:
+
+```Go
+package main
+
+import (
+    "log"
+    "net/http"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+    // Create a span for a web request at the /posts URL.
+    span := tracer.StartSpan("web.request", tracer.ResourceName("/posts"))
+    defer span.Finish()
+
+    // Always keep this trace:
+    span.SetTag(ext.ManualKeep, true)
+
+    // Always drop this trace:
+    span.SetTag(ext.ManualDrop, true)
+}
+```
+
+
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+Manually keep or drop a trace:
+
+```js
+const tags = require('dd-trace/ext/tags')
+
+// To reject the trace
+span.setTag(tags.MANUAL_DROP)
+
+// To keep the trace
+span.setTag(tags.MANUAL_KEEP)
+```
+
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+Manually keep or drop a trace:
+```cs
+using Datadog.Trace;
+
+// Access the active scope through the global tracer (can return null)
+var scope = Tracer.Instance.ActiveScope;
+
+// Always keep this trace
+scope.Span.SetTag(Tags.ManualKeep, true);
```

Comment:
Currently the method signature is `SetTag(string ,string)`, so you can't pass a bool value.

```suggestion
scope.Span.SetTag(Tags.ManualKeep, "true");
```